src/Cake.Core/Scripting/CodeGen/ParameterEmitter.cs

```diff
@@ -0,0 +1,134 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+
+namespace Cake.Core.Scripting.CodeGen
+{
+    /// <summary>
+    /// Responsible for generating parameter tokens in method alias generation
+    /// </summary>
+    internal class ParameterEmitter
```

Comment:
@patriksvensson will by synchronizing my branch against develop this morning; will take care of this request and squash.  thanks!